docs/tutorials/tabular_prediction/tabular-quickstart.md

```diff
@@ -76,20 +78,70 @@ AutoGluon automates this process.
 
-For tabular problems, `fit()` returns a `Predictor` object. Besides inference, this object can also be used to view a summary of what happened during fit.
+For tabular problems, `fit()` returns a `Predictor` object. For classification, you can easily output predicted class probabilities instead of predicted classes:
+
+```{.python .input}
+pred_probs = predictor.predict_proba(test_data_nolab)
+positive_class = [label for label in predictor.class_labels if predictor.class_labels_internal_map[label]==1] # which label is considered 'positive' class
+print(f"Predicted probabilities of class {positive_class}:", pred_probs)
+```
+
+Besides inference, this object can also summarize what happened during fit.
 
 ```{.python .input}
 results = predictor.fit_summary()
 ```
 
-From this summary, we can see that AutoGluon trained many different types of models as well as an ensemble of the best-performing models.  The summary also describes the actual models that were trained during fit and how well each model performed on the held-out validation data.  We can also view what properties AutoGluon automatically inferred about our prediction task:
+From this summary, we can see that AutoGluon trained many different types of models as well as an ensemble of the best-performing models.  The summary also describes the actual models that were trained during fit and how well each model performed on the held-out validation data.  We can view what properties AutoGluon automatically inferred about our prediction task:
 
 ```{.python .input}
 print("AutoGluon infers problem type is: ", predictor.problem_type)
-print("AutoGluon categorized the features as: ", predictor.feature_types)
+print("AutoGluon categorized the features as: ", predictor.feature_types.feature_types_raw)
+special_types = predictor.feature_types.feature_types_special
+if len(special_types) > 0:
+    print("AutoGluon also identified the following special feature types:")
 ```
 
-AutoGluon correctly recognized our prediction problem to be a binary classification task and decided that variables such as `age` should be represented as integers, whereas variables such as `workclass` should be represented as categorical objects.
+AutoGluon correctly recognized our prediction problem to be a **binary classification** task and decided that variables such as `age` should be represented as integers, whereas variables such as `workclass` should be represented as categorical objects.
+
+We can evaluate the performance of each individual trained model on our (labeled) test data:
+```{.python .input}
+model_perf = predictor.leaderboard(test_data, silent=True)
+print(model_perf)
```

Comment:
Use the width special settings like we do in the leaderboard code to ensure this doesn't look ugly. If you look at website, this is not printed nicely like how it is in fit_summary().